corehq/apps/userreports/admin.py

```diff
@@ -21,3 +22,4 @@ class AsyncIndicatorAdmin(admin.ModelAdmin):
 
 
 admin.site.register(AsyncIndicator, AsyncIndicatorAdmin)
+admin.site.register(InvalidUCRData)
```

Comment:
If this lives on the main postgres DB, should we guard against not filling it with lot of data?

Reply:
The idea is that this is an exceptional case. Meaning we should not need to worry about this anymore that we worry about `PillowError` filling up. We could potentially use architect to partition it, but for now it will only be used for ICDS in certain contexts (I don't plan to implement a UI for this in the UCR configs)

Reply:
Yea, I think this shouldn't be a big issue. Even if it get millions of records it wouldn't be that much data.